src/main/java/nl/tudelft/ewi/devhub/server/database/entities/notifications/Notification.java

```diff
@@ -0,0 +1,51 @@
+package nl.tudelft.ewi.devhub.server.database.entities.notifications;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.Date;
+
+/**
+ * Created by Arjan on 7-6-2017.
+ */
+@Data
+@Entity
+@Table(name = "notification")
+@EqualsAndHashCode(of = {"id"})
+public class Notification {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "event")
+    @NotNull
+    //Change this to Enum?
+    private String event;
+
+    @Column(name = "message")
+    @NotEmpty
+    private String message;
+
+    @Column(name = "link")
+    // Change this to URI later
+    private String link;
```

Comment:
I think this can be changed to using an viewable object indeed.

Maybe it is possible to make an interface for this and then use that to store the viewable object?
Because unless we want to store extra information for notifications of different types and don't see the point of creating a separate class for each type.

In this case in the database there is still a field for the type (event) of the notification and a reference to the id of the viewable object.